### ISSUE TITLE
filters do not work in the plug-in while Ajax is being loaded

### ISSUE THREAD

**Issue by makovetskiy** [Nikschavan/unlist-posts#26]
Good day, filters do not work in the plug-in while Ajax is being loaded...

You need to add Ajax support - is_admin () condition should be replaced with (is_admin ()&&! wp_doing_ajax())

**Comment by Nikschavan** [comment]
Can you submit a PR on where this change is needed?

**Comment by makovetskiy** [comment]
https://github.com/Nikschavan/unlist-posts/pull/27

**Comment by Nikschavan** [comment]
Thank you for this @makovetskiy :)

Can you explain any use case for this?
- What exactly is the problem? is this problem caused in case of any plugin?
- How this change fixes it.

I am asking so that I can understand the change better so that it can be maintained correctly in the future.

**Comment by makovetskiy** [comment]
Good day, I created my plugin, and it found such a problem. 1) the Task was to make the button "load more", when you click on the next number of records are loaded at the end of the page, in fact it is ajax-pagination. The problem was this: we have 21 entries, 1 unlist. we have specified 10 records per page, when you first download wordpress uses the filter plugin "unlist-posts" and creates 2 pages of 10 records, when we click on the button "Load more" comes an ajax request in Wordpress each ajax request sets the WP_ADMIN=true, simulating the event from the control panel, is_admin() in any ajax request returns true, and therefore, the filter plug-in "unlist-posts" don't work and pagination broke returning 3 pages instead of 2.
